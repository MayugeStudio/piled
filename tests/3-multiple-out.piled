69 print
420 print
