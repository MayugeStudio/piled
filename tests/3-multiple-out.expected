:int 69
:int 420
